src/features/user/Account/History.tsx

```diff
@@ -60,6 +78,46 @@ export default function History({
     }
   }, [paymentHistory]);
 
+  const handleIssueReceipts = async () => {
+    setLoading(true);
+    try {
+      const res = await postAuthenticatedRequest(
+        '/app/taxReceipts',
+        {
+          year: new Date().getFullYear(),
+        },
+        token,
+        handleError
+      );
+      setLoading(false);
+      if (res && res.length === 0) {
+        setError({
+          type: 'error',
+          message: t('me:taxReceiptsAlreadyGenerated'),
+        });
+        setSelectedRecord(0);
+      } else {
+        await fetchPaymentHistory();
+        setSelectedRecord(0);
+        setOpen(true)
+      }
+    } catch (err) {
+      setLoading(false);
+      console.error(err);
+    }
+  };
+
+  const handleClose = (
+    event?: React.SyntheticEvent | Event,
+    reason?: string
+  ) => {
+    if (reason === 'clickaway') {
+      return;
+    }
+
+    setOpen(false);
+  };
```

Comment:
Perhaps this isn't needed if you use `CustomSnackbar` instead?

Reply:
In MUI Snackbar, the `clickaway` determines whether clicking outside of the Snackbar should dismiss it or not.

If need user to click on the cancel icon on snackbar to close it, and not outside it(might happen by mistake).

If this condition is not present then, clicking outside of the snackbar will close it, and if there is any useful information like https://github.com/Plant-for-the-Planet-org/planet-webapp/pull/1677#discussion_r1105342141 for example, user will lose it, if the click anywhere outside the snackbar.